core/src/main/java/curso/output/RegistrarCursoOutput.java

```diff
@@ -0,0 +1,13 @@
+package curso.output;
+
+import curso.modelo.Curso;
+import curso.modelo.Nivel;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public interface RegistrarCursoOutput {
+    UUID registrarCurso(String nombre, LocalDate fecha, Nivel nivel);
+    //UUID registrarCurso(Curso curso);
```

Comment:
Acá está bien usar esta firma, en lugar de recibir los parámetros
```suggestion
    UUID registrarCurso(Curso curso);
```